disnake/ext/commands/slash_core.py

```diff
@@ -156,20 +160,22 @@ class SubCommandGroup(InvokableApplicationCommand):
     def __init__(
         self,
         func: CommandCallback,
+        parent: InvokableSlashCommand,
```

Comment:
this is technically a breaking change, may want to add that to the changelog just in case

Reply:
Fair, I updated the changelog